lib/bookingsync/api/client/bookings_comments.rb

```diff
@@ -0,0 +1,72 @@
+module BookingSync::API
+  class Client
+    module BookingsComments
```

Comment:
The resource is actually `BookingComment`, not `BookingsComment` (it's not named like join model). Should be `BookingComments` here I think.